.github/workflows/build.yml

```diff
@@ -366,7 +366,7 @@ jobs:
           docker run --name=editoast-test --net=host -v $PWD/output:/output \
             -e DATABASE_URL="postgres://osrd:password@localhost:5432/osrd" \
             ${{ fromJSON(needs.build.outputs.stable_tags).editoast-test }} \
-            /bin/bash -c "diesel migration run --locked-schema && cargo test --verbose -- --test-threads=1 && grcov . --binary-path ./target/debug/ -s . -t lcov --branch --ignore-not-existing --ignore "/*" -o /output/lcov.info"
```

Comment:
Please keep a somewhat low thread limit. In this state, tests fail on my machine (cleanup fails as well).

Reply:
(also applies for README)